src/api/environment.cc

```diff
@@ -408,8 +408,6 @@ void FreeIsolateData(IsolateData* isolate_data) {
   delete isolate_data;
 }
 
-InspectorParentHandle::~InspectorParentHandle() {}
```

Comment:
Why did you remove it here and kept `= default` in other files?

Reply:
I put it [here](https://github.com/nodejs/node/pull/48735/files#diff-7c19dd5fd5584cd4a8111f4c82a9f49f4aa1bdb9470ad11daa90f3d394a23b4eR656-R658)